Lib/test/test_sys.py

```diff
@@ -175,7 +175,8 @@ def test_excepthook_bytes_filename(self):
 
     def test_excepthook(self):
         with test.support.captured_output("stderr") as stderr:
-            sys.excepthook(1, '1', 1)
+            with test.support.catch_unraisable_exception():
+                sys.excepthook(1, '1', 1)
```

Comment:
Why did this need to change?

Reply:
Discussed offline: in debug mode there is a fallback to the C traceback but we still want to see why we are falling back in case there is a problem in the traceback module itself.